src/rootStyles.ts

```diff
@@ -0,0 +1,100 @@
+import { css, CSSResultGroup } from 'lit';
+
+export const rootStyles = css`
```

Comment:
question: Feels like the CSS from `src/index.css` is duplicated here, is there no other way (e.g. load the file in the bundle)?

Reply:
Sadly no, those are the restriction of the shadowDom. I didn't like this either, but there is no either way around it. Shadow DOM is always encapsulated.